src/setup_node_env/root/index.js

```diff
@@ -29,10 +29,8 @@
  */
 
 var force = require('./force')(process.argv);
-
 var uid = process.getuid && process.getuid();
 var isRoot = require('./is_root')(uid);
-
```

Comment:
This isn't a problem, but it's an unrelated change - let's remove it from this PR.